### ISSUE TITLE
ACCURACY about federated_main.py --model=cnn --dataset=cifar --gpu=0 --iid=1 --epochs=10

### ISSUE THREAD

**Issue by Icecream102** [AshwinRJ/Federated-Learning-PyTorch#51]
I copied the command from the author's README.md and I did not change anything, why the accuarcy so low?

 Results after 10 global rounds of training:
|---- Avg Train Accuracy: 30.00%
|---- Test Accuracy: 44.42%

**Comment by monkryeyes** [comment]
I met the same problem. The results of the two federated experiment commands are as follows respectively.
 Results after 10 global rounds of training:
|---- Avg Train Accuracy: 42.00%
|---- Test Accuracy: 45.05%

 Results after 10 global rounds of training:
|---- Avg Train Accuracy: 6.00%
|---- Test Accuracy: 28.99%
